src/rules/at-conditional-rule-no-parentheses/index.js

```diff
@@ -0,0 +1,69 @@
+import { utils } from "stylelint";
+import { namespace } from "../../utils";
+
+export const ruleName = namespace("at-conditional-rule-no-parentheses");
+
+export const messages = utils.ruleMessages(ruleName, {
+  rejected: "Do not use () to surround statements for @-rules"
```

Comment:
This should follow the stylelint naming, so either `Expected ..` or `Unexpected...`